### ISSUE TITLE
How to retraining for 10ms audio

### ISSUE THREAD

**Issue by DyncEric** [breizhn/DTLN#20]
How to retraining this model for 10ms audio frame : ）

**Comment by breizhn** [comment]
Hi,
you have to change self.blockLen and self.block_shift inside the DTLN_model constructor to your desired values or call the properties inside your training script before building the model. For 10ms block length set blockLen to 160 (samples). As block shift 5ms should work, so set it to 80 (samples). The quality is probably similar to the 32 ms model.

**Comment by cloudvc** [comment]
hello there,

I modified the block Len to 160, block shift to 80 and running the training, but the loss is nan...
